roles/archive-upload-ws/tasks/2_install_config.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: ensure archive-upload conf dir exists
+  file:
+    state: directory
+    path: "{{ archive_upload_config_root }}"
+    mode: g+rwXs
+
+#- name: ensure arteria-dsmc log dir exists
+#  file: 
+#    state: directory
+#    path: "{{ arteria_dsmc_log_dir }}"
+#    owner: arteria
+#    group: arteria
+#    mode: g+rwXs
```

Comment:
This can be removed. Old stuff from biotank I think. Log dirs can't be created from within the playbook, as those reside under `/proj/ngi2016001` tree.